### DISCUSSION TITLE
snap data preparation, what is dem_tc.dim ?

### DISCUSSION THREAD

**Discussion by charlottewolff** [insarlab/MintPy#602] [Q&A] [answered]
Hi, 

I am new with mintpy and snap so sorry for my dummy question but I did not find the answer. 
When preparing the data for a processing with snap data, I need to create : 
- dem_tc.dim
- dem_tc.data. 

What is the workflow to create those 2 files ? 

Thanks for your reply, 

Charlotte

**Comment by christinelion** [comment, marked as answer]
Hi Charlotte,

You can see the workflow applied to data processed with SNAP by looking at the "metadata/Processing_Graph" in the Product explorer.
You will notice they use the function add_elevation to have the elevation to the data.
If you have the elevation in your interferogram, you can use "Raster/Bands extractor" to extract the elevation
The dem has to be in the same projection as your interferograms.
The dem can be called elevation_VV.img it worked for me. You need to indicate in your configuration file:
mintpy.load.demFile     = ../dem*/elev*.img

Hope that helps,
Christine

**Comment by charlottewolff** [comment]
Hi Christine ! 
Thanks a lot for your reply. Yes it helps ! I will have a look at the processing graphs. I hope will work. 

May I ask you another question. I tried to apply the snaphu algo to unwrap the interferogram (I tried both : into snap and exporting+using command line) but it stopped with an error... But I dont know what I did wrong.. Do you know at what I have to pay attention when I want to use snaphu ? 

Thanks again for your help 

Charlotte

**Reply by christinelion** [reply]
Hi Charlotte,

For the processes there is also that page https://github.com/insarlab/MintPy/wiki/SNAP-input-data that may help
Just keep in mind you need to merge the IW before the filtering step.

For snaphu you need to make sure you have the tiles overlap each other . You can find some info on the ESA website https://forum.step.esa.int/t/snaphu-unwrapping-error/27045 or here https://forum.step.esa.int/t/snaphu-parameters/8246

Christine
